keda/templates/NOTES.txt

```diff
@@ -58,6 +58,11 @@ WARNING - prometheus.webhooks.serviceMonitor.relabellings is deprecated, please
 WARNING - Running on unsupported Kubernetes version "1.{{.Capabilities.KubeVersion.Minor}}". KEDA 2.11 is supported and tested on Kubernetes "1.25" or higher. See https://keda.sh/docs/2.11/operate/cluster/ for details.
 -------------------------------------------------------------------------------------
 {{- end }}
+{{- if .Values.webhooks.healthProbePort}}
+-------------------------------------------------------------------------------------
+WARNING - webhooks.healthProbePort is deprecated, please migrate to webhooks.livenessProbe instead.
+-------------------------------------------------------------------------------------
+{{- end }}
```

Comment:
`healthProbePort` is also used in the deployment arguments to set the port, I'm not sure if we can deprecate it or if we should use it to override the given port from users. 
What will happen if a user sets 3 probes, all of them with different ports? We need to ensure that it can't happen. Maybe we could instead of using the whole section as yaml, set key by key with the given values

Reply:
Hi @JorTurFer I did a check with the user and his requirement was abit different ([link](https://github.com/kedacore/charts/issues/508#issuecomment-1723479152))
```
operator:
  # Liveness probes
    livenessProbe:
      initialDelaySeconds:  25
      periodSeconds: 10
      timeoutSeconds: 1
      failureThreshold: 3
      successThreshold: 1
  # Readiness probes
    readinessProbe:
      initialDelaySeconds: 20
      periodSeconds: 3
      timeoutSeconds: 1
      failureThreshold: 3
      successThreshold: 1
```
Hence, I have revamped the PR to just address this. Will be much simpler :laughing: When you are free, could you help me look through it too ?

Reply:
****, I misunderstood the commit and I thought that you are overriding the whole section, not just the fields that you have added. my fault, sorry